diff --git i/apps/desktop/src/main/native-messaging.main.ts w/apps/desktop/src/main/native-messaging.main.ts
index 8c8404578b..7d3534dae8 100644
--- i/apps/desktop/src/main/native-messaging.main.ts
+++ w/apps/desktop/src/main/native-messaging.main.ts
@@ -335,7 +335,7 @@ export class NativeMessagingMain {
       return path.join(path.dirname(this.exePath), "resources", "native-messaging.bat");
     }
 
-    return this.exePath;
+    return "/usr/bin/bitwarden-desktop";
   }
 
   private getRegeditInstance() {
